src/citrine/resources/predictor.py

```diff
@@ -75,7 +75,7 @@ def check_for_update(self, uid: Union[UUID, str] = None,
         else:
             return None
 
-    def auto_configure(self, *, training_data: DataSource, pattern="PLAIN") -> Predictor:
+    def auto_configure(self, *, training_data: DataSource, pattern="INFER") -> Predictor:
```

Comment:
Why is changing the default behavior motivated?  In theory, this could be breaking and we aren't supposed to make any breaking changes until 2.0.  Is this feature still in Alpha?

Reply:
I'm okay with the above rational.  Sounds like this is not breaking a workflow.